main/EventSystem.cpp

```diff
@@ -45,10 +45,25 @@ static std::string m_printprefix;
 extern PyObject * PDevice_new(PyTypeObject *type, PyObject *args, PyObject *kwds);
 #endif
 
+typedef enum
```

Comment:
Is it possible you can follow our ENUM declararions ? If you search in our project for 'typedef enum' you will see what i mean ... not like 'tString' ?
Like JTYPE_STRING, JTYPE_FLOAT
Missed this one before...